Readme.md

```diff
@@ -36,24 +77,14 @@ and does not contain the actual implementation.
 
 The implementation mode requires defining the preprocessor macro
 `NK_IMPLEMENTATION` in *one* .c/.cpp file before `#include`ing this file, e.g.:
-```c
-#define NK_IMPLEMENTATION
-#include "nuklear.h"
-```
-IMPORTANT: Every time you include "nuklear.h" you have to define the same optional flags.
-This is very important; not doing it either leads to compiler errors, or even worse, stack corruptions.
 
-## Gallery
+**IMPORTANT:** Every time you include "nuklear.h" you have to define the same optional flags.
+This is very important; not doing it either leads to compiler errors, or even worse, stack corruptions.
```

Comment:
Could have this as a Trouble Shooting section in the docs folder or something. It's not really worth mentioning in a Quick Start section.

Reply:
This was a frequent issue for newcomers back in the past - therefore it made it to readme :wink:. Adding it to a Trouble Shooting section is a good idea but nonetheless we shall at least keep somewhere here a big warning referencing that particular trouble shooting point.

Reply:
Seems fine to mention here if it's a common issue 😉